test/litmus/type/string_test.exs

```diff
@@ -112,6 +152,21 @@ defmodule Litmus.Type.StringTest do
       assert Litmus.validate(data, schema) ==
                {:error, "#{field} length must be #{length} characters"}
     end
+
+    test "errors when the value is nil" do
```

Comment:
Is there a case in which `length: 0`?

Reply:
i guess that's possible. And you're saying `nil` should be valid if `length: 0`?

Reply:
Nvm talked IRL, all is good